src/Root.tsx

```diff
@@ -0,0 +1,23 @@
+import {
+  Route,
+  Routes,
+  Navigate,
+  HashRouter as Router,
+} from 'react-router-dom';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { App } from './App';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
```

Comment:
You can add all paths in the enum and use them in all places in your application.